src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -439,6 +443,27 @@ private void setupToolBar() {
 
         toolBar.add(writeXmp);
 
+        JPopupMenu fetcherPopup = new JPopupMenu();
+        for(EntryBasedFetcher fetcher : EntryFetchers.getEntryBasedFetchers()) {
+            fetcherPopup.add(new JMenuItem(new AbstractAction(fetcher.getName()) {
+
+                @Override
+                public void actionPerformed(ActionEvent e) {
+                    new EntryFetchAndMergeWorker(panel, getEntry(), fetcher).execute();
+                }
+            }));
+        }
+        JButton fetcherButton = new JButton(IconTheme.JabRefIcon.REFRESH.getIcon());
+        fetcherButton.setToolTipText(Localization.lang("Update with bibliographic information from the web."));
```

Comment:
Imho, there is no reason for adding the period in the end. JabRef is not consistent in this, some tool tips end with periods and some do not. In my point of view, punctuation is only needed if there is a block of text for the tool tip. If it is a single sentence, no punctuation looks better.

So please remove that one :)